### PR TITLE
[improve][doc] Add configurations doc for the python client

### PR DESCRIPTION
**Pull request by RobertIndie**
Here are specific links for these configuration doc:
* Client: https://pulsar.apache.org/api/python/3.1.x/pulsar.Client.html#__init__
* Producer: https://pulsar.apache.org/api/python/3.1.x/pulsar.Client.html#create_producer
* Consumer: https://pulsar.apache.org/api/python/3.1.x/pulsar.Client.html#subscribe
* Reader: https://pulsar.apache.org/api/python/3.1.x/pulsar.Client.html#create_reader 

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: docs/client-libraries-python-configs.md [hunk 1]
```diff
@@ -4,4 +4,18 @@ title: Pulsar Python client configurations
...
 
-Coming soon...
\ No newline at end of file
+## Client configuration
+
+For all available configurations, see [`the constructor of the pulsar client`](@pulsar:apidoc:python@/pulsar.Client.html#__init__).
+
+## Producer configuration
+
+For all available configurations, see [`client.create_producer`](@pulsar:apidoc:cpp@/pulsar.Client.html#create_producer).
```
Comment (tisonkun) [inline docs/client-libraries-python-configs.md:13]: cpp?

File: docs/client-libraries-python-configs.md [hunk 2]
```diff
@@ -4,4 +4,18 @@ title: Pulsar Python client configurations
...
+
+## Consumer configuration
+
+For all available configurations, see [`client.subscribe`](@pulsar:apidoc:cpp@/pulsar.Client.html#create_producerl).
```
Comment (momo-jun) [inline docs/client-libraries-python-configs.md:17]:
```suggestion
For all available configurations, see [`client.subscribe`](@pulsar:apidoc:python@/pulsar.Client.html#subscribe).
```

File: docs/client-libraries-python-configs.md [hunk 3]
```diff
@@ -4,4 +4,18 @@ title: Pulsar Python client configurations
...
+
+## Reader configuration
+
+For all available configurations, see [`client.create_reader`](@pulsar:apidoc:cpp@/pulsar.Client.html#create_producer).
```
Comment (momo-jun) [inline docs/client-libraries-python-configs.md:21]:
```suggestion
For all available configurations, see [`client.create_reader`](@pulsar:apidoc:python@/pulsar.Client.html#create_reader).
```

File: docs/client-libraries-python-configs.md (hunk 1 above)
Comment (momo-jun) [inline docs/client-libraries-python-configs.md:13]:
```suggestion
For all available configurations, see [`client.create_producer`](@pulsar:apidoc:python@/pulsar.Client.html#create_producer).
```

**Review by tisonkun — APPROVED** [review]
LGTM

**Review by momo-jun — APPROVED** [review]
Thanks for adding the docs!

**Pull request merged**